### DISCUSSION TITLE
SS2K always comes up in access point mode

### DISCUSSION THREAD

**Discussion by rjbutler** [doudar/SmartSpin2k#306] [Q&A] [answered]
@doudar, after the last version update the SS2K seems to be stuck in access point mode.  I enter my SSID and password, save, reboot and it always comes up advertising SmartSpin2K instead of connecting the the access point sitting 10 feet away.  

Any ideas?

**Comment by rjbutler** [comment, marked as answer]
I built 1.12.30 and flashed via VSCode and that seems to have resolved it.

**Comment by doudar** [comment]
Glad you were able to fix it. 

The only thing I can think of is if for some reason the SPIFFS is full and it was unable to write the config.txt to the partition. Currently the only time I've seen this is if a large file was uploaded to SPIFFS via the manual firmware update page. 

For anyone else that encounters this problem, a "reset to defaults" should also fix the issue as that will completely reformat the SPIFFS partition.
